pkg/cmd/profile/setdefault/setdefault.go

```diff
@@ -28,7 +28,7 @@ func NewSetDefaultCmd(f *cmdutil.Factory, runF func(*SetDefaultOptions) error) *
 	}
 	cmd := &cobra.Command{
 		Use:               "setdefault <profile>",
-		Args:              validators.ExactArgs(1),
+		Args:              validators.ExactArgsWithDefaultRequiredMsg(1),
```

Comment:
On the naming, I kind of changed my mind.
Could we do instead the `validators.ExactArgs(1)` to have the default required messaging, and a `validators.ExactArgsWithMessage(1, msg)` maybe? IMHO it will more elegant.

Reply:
I agree too since we use the default message everywhere 👍